torch/jit/_monkeytype_config.py

```diff
@@ -16,6 +15,25 @@
 except ImportError:
     _IS_MONKEYTYPE_INSTALLED = False
 
+def get_type(type):
+    """
+    Helper function which converts the given type to a torchScript acceptable format.
+    """
+    if inspect.getmodule(type) == typing:
+        # If the type is a type imported from typing
+        # like Tuple, List, Dict then replace `typing.`
+        # with a null string. This needs to be done since
+        # typing.List is not accepted by TorchScript.
+        type_to_string = str(type)
+        return type_to_string.replace(type.__module__ + '.', ' ') + ','
+    elif 'torch' in type.__module__:
+        # If the type is a subtype of torch module, then TorchScript expects a fully qualified name
+        # for the type which is obtained by combining the module name and type name.
+        return type.__module__ + '.' + type.__name__ + ','
+    else:
+        # For all other types use the name for the type.
+        return type.__name__ + ','
```

Comment:
It is bug-prone and non-Pythonic to construct a comma separated list this way. A popular alternative is to construct a list and then join then together.